### DISCUSSION TITLE
Windows warning for installer 6.3.2

### DISCUSSION THREAD

**Discussion by flommmm** [hluk/CopyQ#2186] [Q&A]
Hi,

when I try to install version 6.3.2 (windows 11, exe-file), I get this error:

TRANSLATION:
The computer was protected by windows. The start of an unknown app was prevented by microsoft defender smartscreen. The execution may contain a risk for the PC. 
App: copyq-6.3.2-setup.exe
Publisher: Unknown publisher

_ORIGINAL MESSAGE (GERMAN):
Der Computer wurde durch Windows geschützt
Von Microsoft Defender SmartScreen wurde der Start einer unbekannten App verhindert. Die Ausführung stellt u.U. ein Risiko für den PC dar.
App: copyq-6.3.2-setup.exe
Herausgeber: Unbekannter Herausgeber_

I did not get this message when I installed Version 6.1.0 and our IT recommends not installing it. Any idea on this?

Best regards
flommmm

**Comment by hluk** [comment]
See #2183.
